pbft/pbft.cpp

```diff
@@ -1571,7 +1665,7 @@ pbft::broadcast_new_configuration(pbft_configuration::shared_const_ptr config)
 }
 
 bool
-pbft::is_configuration_acceptable_in_new_view(hash_t config_hash)
+pbft::is_configuration_acceptable_in_new_view(const hash_t& config_hash)
 {
     return this->configurations.is_enabled(config_hash);
```

Comment:
we should accept a configuration for a new view if we've accepted its preprepare; I think is_enabled corresponds to the operation being prepared, which is a stricter requirement.